test/parse_test.rb

```diff
@@ -1117,7 +1174,10 @@ class ParseTest < Test::Unit::TestCase
       IDENTIFIER("foo"),
       PARENTHESIS_LEFT("("),
       ArgumentsNode(
-        [StarNode(IDENTIFIER("rest"), CallNode(nil, nil, IDENTIFIER("rest"), nil, nil, nil, "rest"))]
+        [StarNode(
+          STAR("*"),
```

Comment:
not quite related, but operator should be a star, not `rest` here, right? cause we have `rest` in the call node

Reply:
Yeah this is definitely incorrect as it currently stands

Reply:
@kddnewton cool, resolved conflicts, ready for a next round of review 🙂